pulsar-transaction/coordinator/src/main/proto/PulsarTransactionMetadata.proto

```diff
@@ -51,6 +51,7 @@ message TransactionMetadataEntry {
   optional uint64 start_time      = 9;
   optional uint64 last_modification_time = 10;
   optional uint64 max_local_txn_id = 11;
+  optional string owner          = 12;
```

Comment:
I've added an answer in the doc's comment. I'm reporting it here

> 
> I'm focusing on "If users don't use the exclusive transactions" since it will be the common use case anyway.
> 
> We can't use the transaction-key as authorization data source because it enforces the admin to allow single users to specific transactions key and adds operational costs.
> From a user's perspective, this is very annoying. 
> 
> Most of the cases would end up in use the default transaction-key (which I don't know how it could be generated). I'm not sure how you could have default keys without impacting other clients in the same namespace and fencing them out unintentionally. 
> 
> 
> Adding in future the transactions auth permission makes sense if you're going to use the per-namespace transactions. This will be based on the current auth model, which is well-known to Pulsar's users.
> 
> 
> Also, for the impact of the storage:
> 1. if we store this "default transaction key" that means you are not using exclusive transactions but you're going to store that anyway.
> 2. Exclusive transactions might have a global flag for enable/disable them. How you're going to deal with authorization then?
> 
> I believe the storage overhead (in case of exclusive transactions) will be acceptable and it helps in making Pulsar more user friendly